bin/jobsub_cmd

```diff
@@ -49,8 +49,12 @@ class StoreGroupinEnvironment(argparse.Action):
         setattr(namespace, self.dest, values)
 
 
+arglist = argparse.Namespace()
```

Comment:
So this applies to all four files, but what would you think about _just_ making the verbose value a global?  Since we only need that one variable globally, I'm thinking something along the lines of:

For this line:

```python
verbose = 0
```
Then, down on line 57, it'd be

```python
global verbose
```

And then, after you parse the args on line 71, you could set the global value like:

```python
verbose = args.verbose
```

And down on 240:
```python
if verbose:
    raise
```

There might even be a cleaner way than that, but it's just a thought that occurred to me.

Reply:
I like that.   Done.